src/main/java/org/cascadebot/cascadebot/commandmeta/CommandContext.java

```diff
@@ -300,42 +239,7 @@ private Argument getArgFromSet(Set<Argument> arguments, String arg) {
     }
 
     public void sendPermissionsError(String permission) {
```

Comment:
Move this into MessagingUI and have `Permission permission` as the parameter instead

Reply:
I almost think these sould be in another category, but i don't know what we'd call it, and it wound;t have much.

Reply:
I don't think there's a huge need for it, UI makes sense to me